tasks/run-e2e

```diff
@@ -144,12 +167,22 @@ const args = yargs
 
 const REDWOODJS_FRAMEWORK_PATH = path.resolve(__dirname, '..')
 let REDWOOD_PROJECT_DIRECTORY =
-  args._?.[0] || path.join(os.tmpdir(), 'redwood-e2e', new Date().toISOString())
+  args._?.[0] ||
+  path.join(
+    os.tmpdir(),
+    'redwood-e2e',
+    // ":" is problematic with paths
+    new Date().toISOString().split(':').join('-')
+  )
 
 console.log()
 console.log('-'.repeat(80))
 console.log()
-makeDirectory(REDWOOD_PROJECT_DIRECTORY)
+if (args._?.[0]) {
+  console.log('Using supplied directory \n', REDWOOD_PROJECT_DIRECTORY)
+} else {
+  makeDirectory(REDWOOD_PROJECT_DIRECTORY)
+}
```

Comment:
The intention here is that a user can also supply a path. If the path doesn't exist it'll be created, but if it does then nothing happens.

Super helpful when trying to debug something:
`rm -rf /tmp/rwjs-test; ./tasks/run-e2e; cat /tmp/rwjs-test/web/src/App.tsx`

Reply:
Ah gotcha. Will remove the block.

Reply:
Done